README.md

```diff
@@ -23,26 +23,26 @@ HTTP API
 Installation
 ------------
 
-The best way to run Puppet is with [Puppet Enterprise](https://puppet.com/puppet/puppet-enterprise),
+The best way to run Puppet is with [Puppet Enterprise (PE)](https://puppet.com/puppet/puppet-enterprise),
 which also includes orchestration features, a web console, and professional support.
-[The PE documentation is available here.](https://puppet.com/docs/pe/latest)
+The PE documentation is [available here.](https://puppet.com/docs/pe/latest)
 
 To install an open source release of Puppet,
-[see the installation guide on the docs site.](http://puppet.com/docs/puppet/5.4/install_pre.html)
+[see the installation guide on the docs site.](http://puppet.com/docs/puppet/latest/install_pre.html)
```

Comment:
We were trying to pin this version number to the current release because "latest" will always point to the most recently released version of Puppet, which might not be relevant if someone is viewing the source of or using an older release.

This also affects the Ruby API docs generated from YARD, which was resulting in issues with the copy of those docs we were hosting on puppet.com/docs, but I think we're still phasing those out.